src/components/LetterGrid/LetterGrid.tsx

```diff
@@ -0,0 +1,85 @@
+'use client';
+
+import { useState } from 'react';
+import styles from './letterGrid.module.scss';
+
+const classNames = require('classnames');
+
+export const LetterGrid = () => {
+  const [wordInput, setWordInput] = useState('');
+
+  const handleUserInput = (id: string) => {
+    const chosenLetter = document.getElementById(id);
```

Comment:
As React uses the virtual DOM, we should not/ cannot access the DOM directly. Even if this seem to work, this is definitely unreliable. Virutal DOM manages the state and then updates the DOM. Here we try to access a DOM element that might not have been updated yet.